app/controllers/api/v1/volunteers_controller.rb

```diff
@@ -0,0 +1,57 @@
+require 'net/http'
+module Api
+  module V1
+    class VolunteersController < ApplicationController
+      include Response
```

Comment:
The find_volunteers method should be added as a before filter method here instead of being used in the update and destroy methods. It's best practice to have `before_action :find_volunteer, only: %![update destroy]`